lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -38,7 +38,7 @@
 }
 
 // Add a long task to delay FI
-setTimeout(longTask, 2000);
```

Comment:
why this change?

Reply:
I noticed that when I originally added the `loading=lazy` test, it passed the `offscreen` audit even without my change. @paulirish helped me figure out why: it turned out that the image with native lazy loading was being filtered out [here](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/byte-efficiency/offscreen-images.js#L147). When we lengthened the below long task, the lazy loaded image request will load before TTI, so we were able to test the desired code path. Given that it depends on relative timing, it's not totally ideal, so would be open to learning if there's another way!

Reply:
Gotcha, that makes sense! There's not really a great way to isolate that in the smoketests without taking much, much longer than they do now unfortunately, so this will do! 👍